### PR TITLE
no need to expose port for probes

### PR DESCRIPTION
**Pull request by NargiT**
I was doing the exercise and there is no need to expose the container port on kubernetes v1.15.1

### CODE REVIEW

**Comment by dgkanatsios** [comment]
Thanks for your contribution!
I'd rather have a comment there explaining that it is optional, rather than delete it completely. Let's maintain backwards compatibility!

**Comment by NargiT** [comment]
hopefully it's more clear

File: e.observability.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ spec:
     name: nginx
     resources: {}
     ports:
-      - containerPort: 80
+      - containerPort: 80 # Note: Readiness probes runs on the container during its whole lifecycle. Since nginx exposes 80, containerPort: 80 is not requierd for readiness to work.
```
Comment (dgkanatsios) [inline e.observability.md]: typo: should be `required`

**Comment by dgkanatsios** [comment]
thanks! minor typo and we can merge!

**Comment by NargiT** [comment]
done

**Pull request merged**